tiny_cnn/core/nnp_backend.h

```diff
@@ -117,6 +114,10 @@ class nnp_backend : public math_backend {
           threadpool,
           profile);
 
+      if (status != nnp_status_success) {
+          throw nn_error("Could not succeed with nnp_convolution_inference");
+      }
+
       // TODO: embed it into a class
```

Comment:
@nyanp The result is the same.

@bhack @mtamburrano 
Besides, I added a status checking and I realized that it raises an error each time. Maybe we are not configuring well NNPACK. I'll squash my commits and after that you can merge the PR.